src/sentry/coreapi.py

```diff
@@ -271,9 +272,7 @@ def safely_load_json_string(self, json_string):
             raise APIError('Bad data reconstructing object (%s, %s)' %
                 (type(e).__name__, e)
             )
-
-        # XXX: ensure keys are coerced to strings
-        return dict((smart_bytes(k), v) for k, v in six.iteritems(obj))
```

Comment:
this is a bit concerning, but i dont know why we were doing this unless it was for python 2.6 compat